webdriver/tests/bidi/browsing_context/close/prompt_unload.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+import asyncio
+
+from webdriver.bidi.modules.script import ContextTarget
+
+pytestmark = pytest.mark.asyncio
+
+CONTEXT_DESTROYED_EVENT = "browsingContext.contextDestroyed"
+USER_PROMPT_OPENED_EVENT = "browsingContext.userPromptOpened"
+
+
+@pytest.mark.parametrize("type_hint", ["window", "tab"])
+@pytest.mark.parametrize("prompt_unload", [None, False])
+async def test_prompt_unload_not_triggering_dialog(bidi_session, subscribe_events, setup_beforeunload_page,
+                                                   wait_for_event, wait_for_future_safe, type_hint, prompt_unload):
```

Comment:
nit: similar to other tests I got the following as suggestion from black (would also apply to the other test in this file):

```suggestion
async def test_prompt_unload_not_triggering_dialog(
    bidi_session, 
    subscribe_events, 
    setup_beforeunload_page,
    wait_for_event, 
    wait_for_future_safe, 
    type_hint, 
    prompt_unload
):
```